start.sh

```diff
@@ -240,4 +240,4 @@ startCloudflare
 
 echo
 echo 'Touchbase cluster is launched!'
-echo "Try scaling it up by running: ./start ${CONFIG_FILE} scale"
+echo "Try scaling it up by running: ./start.sh ${CONFIG_FILE} scale"
```

Comment:
`-p` is set by default in our `start.sh` script.

The `-f` is elided as a result of me being overly clever with the opt parsing so that you don't need to pass `-f` if you want the default. Lemme see if I can clean this up a bit.